main_app/templates/main_app/event_list.html

```diff
@@ -1,84 +1,128 @@
-{% extends 'base.html' %}
-{% block content %}
+{% extends 'base.html' %} {% block content %}
 
 <div class="container">
   <div class="row valign-wrapper">
     <div class="col s12 m8">
-        <h1>Cosmic Events</h1>
+      <h1>Cosmic Events</h1>
     </div>
     <div class="col s12 m4 right-align">
-        <a class="dropdown-trigger btn" href="#" data-target="filter-dropdown"><span class="material-icons" style="vertical-align:middle; margin-right: 5px;">sort</span>Sort</a>
-        <ul id="filter-dropdown" class="dropdown-content">
-            <li><a href="?orderby=title">By Name</a></li>
-            <li><a href="?orderby=start_date">By Date</a></li>
-            <li><a href="?orderby=event_type">By Event Type</a></li>
-            <li><a href="?orderby=users_watching">By # of Watchers</a></li>
-        </ul>
+      <a class="dropdown-trigger btn" href="#" data-target="filter-dropdown"
+        ><span
+          class="material-icons"
+          style="vertical-align: middle; margin-right: 5px"
+          >sort</span
+        >Sort</a
+      >
```

Comment:
@mayaellington I think your VS Code may have been acting up and adding line breaks. Not sure if these will cause issues, but I think it's best if we clean them up before merging.

Reply:
I fixed that line, must have been VS code formatting, weird!